public/examples/integrations/toolkits/reddit/comment_on_post_example_call_tool.js

```diff
@@ -0,0 +1,33 @@
+import { Arcade } from "@arcadeai/arcadejs";
+
+const client = new Arcade(); // Automatically finds the `ARCADE_API_KEY` env variable
+
+const USER_ID = "you@example.com";
+const TOOL_NAME = "Reddit.CommentOnPost";
+
+// Start the authorization process
+const authResponse = await client.tools.authorize({
+  tool_name: TOOL_NAME,
+  user_id: USER_ID,
+});
+
+if (authResponse.status !== "completed") {
+  console.log(`Click this link to authorize: ${authResponse.url}`);
+}
+
+// Wait for the authorization to complete
+await client.auth.waitForCompletion(authResponse);
+
+// TODO: Replace with an identifier for a real post
+const toolInput = {
+  post_identifier: "https://www.reddit.com/r/TestSubreddit/comments/1abcdefg/",
+  text: "This is a test comment on a non-existent subreddit."
+};
+
+const response = await client.tools.execute({
+  tool_name: TOOL_NAME,
+  input: toolInput,
+  user_id: USER_ID,
+});
+
+console.log(response);
```

Comment:
I think we need to start doing `console.log(response.output.value);` in all examples (inc Python)

logging the `response` doesn't output the actual content, but an object. in order to print the actual tool response, we need to reference `response.output.value`.